docs/testing/releases/380.md

```diff
@@ -0,0 +1,21 @@
+## Testing notes and ZIP for release 3.8.0
+
+Zip file for testing: [woocommerce-gutenberg-products-block.zip](https://github.com/woocommerce/woocommerce-gutenberg-products-block/files/5516903/woocommerce-gutenberg-products-block.zip)
+
+## The following changes affect the feature plugin only.
+
+### Show the phone number field in the billing section when shipping is disabled in settings. ([3376](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/3376))
+
+Moves the rendering of address fields to a single file so that PhoneNumber (new component) can be rendered in either billing or shipping, depending on visibility, without duplication.
+
+0. Ensure the phone number field is enabled in block settings, then test 2 cases:
+1. With shipping enabled, ensure phone number displays beneath the shipping step.
+2. With shipping disabled (you can delete all rates and zones to edit general settings), ensure phone number displays beneath the billing step.
+3. In both scenarios the values should change and be saved as billing information.
```

Comment:
> be saved as billing information

Is there a specific way to verify that? I went to WooCommerce > Orders to make sure the phone value was correct.  Might be good to specify the exact steps.

Reply:
Got it. Yes, it can be misleading. Since there is only one way to save the date from the code perspective if the phone number is saved then it is OK. We only change the presentation on the backend if it is saved then it is saved. I will simplify the instruction.